gameofthrones/characters/Daenerys.java

```diff
@@ -0,0 +1,15 @@
+public class Daenerys {
+
+    private List<String> titles = new ArrayList<String>();
+    titles.add("Queen of the Andals and the First Men");
```

Comment:
Good, is better do this in a constructor.  What do you think?

Reply:
I think there is always so many titles that even a constructor is going to be limited. I like this approach but a better way of storing the titles should be found.

Reply:
ok, this aproach is valid. Would be nice have a method or a constructor to initialize this variables, thinking of the scalability of the class. Thank you for your answer.